test/graphql/queries/profile_query_test.rb

```diff
@@ -285,8 +285,12 @@ class ProfileQueryTest < ActiveSupport::TestCase
     profile1_result = result['data']['allProfiles'].find do |h|
       h['id'] == profiles(:one).id
     end
+    profile2_result = result['data']['allProfiles'].find do |h|
+      h['id'] == profiles(:two).id
+    end
     assert_equal policies(:one).name, profile1_result['name']
-    assert_equal 3, profile1_result['totalHostCount']
+    assert_equal 2, profile1_result['totalHostCount']
+    assert_equal 2, profile2_result['totalHostCount']
```

Comment:
And, as expected, no matter which profile queried for `totalHostCount`, the result is the same.